index.html

```diff
@@ -24,126 +41,612 @@
         console.log('polyfill applied');
       }
     </script>
+    <script src="https://unpkg.com/prismjs@v1.x/components/prism-core.min.js"></script>
+    <script src="https://unpkg.com/prismjs@v1.x/plugins/autoloader/prism-autoloader.min.js"></script>
+
+    <!-- TypeKit Fonts -->
+    <script src="https://use.typekit.net/slx1xnq.js"></script>
+    <script>
+      try {
+        Typekit.load({ async: true });
+      } catch (e) {}
+    </script>
   </head>
 
   <body>
-    <h1>Popover Attribute Polyfill</h1>
-
-    <div id="popovers">
-      <div id="popover1" popover>Popover 1</div>
-      <div id="popover2" popover="">Popover 2</div>
-      <div id="popover3" popover="auto">
-        Popover 3. This popover has an autofocus link:
-        <a href="#" autofocus>I'm a link!</a>
-      </div>
-      <div id="popover4" popover="hint">Invalid Popover ("hint")</div>
-      <div id="popover5" popover="manual">
-        Popover 5. This popover has an autofocus link:
-        <a href="#" autofocus>I'm a link!</a>
-      </div>
-      <div id="popover6" popover="invalid">Invalid Popover ("invalid")</div>
-      <div id="popover7" popover="auto">Popover 7 (auto)</div>
-      <div id="popover8" popover="auto">Popover 8 (auto)</div>
-      <div id="popover9" popover="manual">Popover 9 (manual)</div>
-      <div id="popover10" popover="manual">Popover 10 (manual)</div>
-      <div id="popover11" popover="auto">
-        <ul>
-          <li><button>Some</button></li>
-          <li><button>Menu</button></li>
+    <header class="site-header">
+      <a href="/" rel="home" id="banner-logo">
```

Comment:
Should this link to the oddbird site? It's not clear to me how the two should integrate, but that's what I expected - especially since the demo is a single page.